core/policy/provider.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
```

Comment:
Once a new file is created, the date should be set to the current year. 
If you edit the file the date should not be changed. 
```suggestion
 * Copyright (C) 2023 The "MysteriumNetwork/node" Authors.
```